src/expression.c

```diff
@@ -6961,6 +6961,25 @@ Expression *FileExp::semantic(Scope *sc)
     se = se->toUTF8(sc);
     name = (char *)se->string;
 
+    if (name[0] == ':')
```

Comment:
This works with 0-terminated strings, but with D strings, it will throw a `RangeError` if `name` is empty. Does MagicPort takes this into account?